src/main/java/core/basesyntax/ConsoleHandler.java

```diff
@@ -0,0 +1,174 @@
+            "help", "help [command]\n"
+                    + "Get info about all available commands or about a specified one.\n",
+            "exit", "exit\nSay bye-bye :-)\n");
+
+    public void handleInput() throws IOException {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String input = scanner.nextLine();
+            String[] args = input.split(" ");
+            String pathOrCommandName = null;
+            String fileName = null;
+            if (args.length >= 2) {
+                pathOrCommandName = args[1];
+                fileName = args[2];
+            }
+            switch (args[0]) {
+                case "create":
+                    create(pathOrCommandName, fileName);
+                    break;
+                case "read":
+                    read(pathOrCommandName, fileName);
+                    break;
+                case "info":
+                    getInfo(pathOrCommandName, fileName);
+                    break;
+                case "help":
+                    help(args);
+                    break;
+                case "exit":
+                    System.exit(0);
+                    break;
+                default:
+                    saveText(input);
+            }
+        }
+    }
+
+    private void getInfo(String path, String fileName) throws IOException {
+        Path file = Path.of(path + File.separator + fileName);
+        System.out.println(Files.readAllBytes(file).length + " symbols");
+        System.out.println(Files.readAllLines(file).size() + " lines");
+        System.out.println(countWords(Files.readAllLines(file)) + " words");
+        BasicFileAttributes attributes = Files.readAttributes(file, BasicFileAttributes.class);
+        DateFormat df = new SimpleDateFormat("dd-MM-yyyy");
+        String dateLastModified = df.format(attributes.lastModifiedTime().toMillis());
+        System.out.println(dateLastModified + " last modified");
+        System.out.println("Size: " + attributes.size() + " bytes");
+    }
+
+    private long countWords(List<String> lines) {
+        // this is a hw for lesson 5, so students are not obliged to use Stream API
+        return lines.stream()
+                .map(line -> Stream.of(line.split(" ")))
+                .flatMap(o -> o)
+                .filter(str -> str.matches("^[a-zA-Z]+"))
+                .count();
+    }
+
+    private void help(String[] args) {
+        if (args.length >= 2) {
+            System.out.println(commandInfo.get(args[1]));
+            return;
+        }
+        for (String info : commandInfo.values()) {
+            System.out.println(info);
+
+        }
+
+    }
+
+    private void saveText(String input) throws IOException {
+        if (input.isEmpty()) {
+            return;
+        }
+        System.out.println("Save text? Y/N");
+        if (askYesNo()) {
+            String[] filePath = askUserFilePath();
```

Comment:
my comment was not fixed

we are using data structure which is not meant to store what we are storing. instead of returning array of two elements, we should ask use separately to enter file name and path. And store them separately.

Reply:
thank you for a detailed explanation :)